### DISCUSSION TITLE
how to install theia in vultr cloud ?

### DISCUSSION THREAD

**Discussion by Lashely** [eclipse-theia/theia#9507] [Q&A]
i need my personal ide in vultr can it possible ?

**Comment by vince-fugnitto** [comment]
@Lashely thank you for the discussion, it should be possible however there is no official guide or documentation on how to do so.
I've seen two references in the past to others using `vultr` and `theia` together:
- https://community.theia-ide.org/t/experiences-adopting-theia/592/2
- https://github.com/eclipse-theia/theia/issues/2714#issue-355581165
